template/iconComponent.ts

```diff
@@ -0,0 +1,31 @@
+interface templateProps {
+    fileName: string;
+    data: string;
+}
+
+function componentTemplate ({fileName,data}:templateProps) {
+    const render = `
+// GENERATED BY ./scripts/iconScript.ts
+
+import * as React from 'react';
+
+interface iconProps {
+height?: string | number | undefined,
```

Comment:
`height?` itself means its not required, no need to give undefined as a type

Reply:
are we using height and width both as a prop or going with just `size` prop? @en3sis what do you think?

Reply:
also why do we have to specify interface for props in every icon component? we can just specify it at once place and import it in the template right?

Reply:
> also why do we have to specify interface for props in every icon component? we can just specify it at once place and import it in the template right?

sure @kbhutani0001, I will create a separate file for the interface and call the interface for the props from that file.

Reply:
I have added a file named helper.ts from which all the components will import the interface.

Reply:
@kbhutani0001 how about we accept a single variable for size and use it for both width and height
```react
<svg height={size} width={size} .... >
```

Reply:
all of our icons are square (have viewBox="0 0 24 24"). we need to scale either one of these using size. Not sure if we need both of these. @en3sis can suggest more on this

Reply:
Sorry I totally missed this mention email =/

I agree with Kartikay, I don't think we need different sizes maybe we could do is something like
`size={24}` and use this as both `width` and `height` value.

I was also thinking to have pre-defined aliases, ex:

`
xs: 4,
s: 8,
base: 14, // we need to think about this, something to have in between
m: 18,
l: 24,
xl: 32,
xxl: 48,
xxxl: 64,
`

So we could allow users to pass the text value, ex: `<....  size="xl" />` or as custom size, `< ... size='200' />.
If the size prob is not defined, we should return a default (`m` or `base`)

Reply:
@en3sis I have added the option to pass strings for using the above pre-defined aliases. It still supports numbers as size value. 
Removed the two props for height and width, instead kept a single prop size for adjusting the size.
Also in case the user isn't providing a size, it will default to "m" - 18.